.travis.yml

```diff
@@ -67,10 +66,6 @@ before_script:
 
 script:
     - composer validate --strict
-    - vendor/bin/phpstan analyse -c phpstan.neon -l max src/
-
-    - vendor/bin/phpunit
```

Comment:
We removed it to pass tests? ;>

Reply:
We don't have any unit tests so I've decided to remove it, also in other plugins, we don't run PHPstan analysis. If we want to then we must make deeper refactoring.